### PR TITLE
Tuning lighting for system(s) with/without GPU

### PR DESCRIPTION
**Pull request by wangvnn**
- Adjust ceiling lamp's parameters so that it works system(s) with/without GPU

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: worlds/no_roof_small_warehouse.world [hunk 1]
```diff
@@ -224,7 +224,7 @@
       <specular>0.2 0.2 0.2 1</specular>
       <attenuation>
         <range>80</range>
-        <constant>0.01</constant>
+        <constant>0.3</constant>
```
Comment (ojasjoshi) [inline worlds/no_roof_small_warehouse.world:227]: This attenuation value is tuned for CPU or GPU?

File: README.md [hunk 2]
```diff
@@ -67,3 +67,6 @@ roslaunch aws_robomaker_small_warehouse_world view_small_warehouse.launch
 
 **Visit the [AWS RoboMaker website](https://aws.amazon.com/robomaker/) to learn more about building intelligent robotic applications with Amazon Web Services.**
 
+## Notes
+- Lighting might vary on different system(s) (e.g brighter on system without CPU and darker on system with GPU)
```
Comment (ojasjoshi) [inline README.md:71]: Is this an ideal behavior?

File: worlds/no_roof_small_warehouse.world (hunk 1 above)
Comment (wangvnn) [reply worlds/no_roof_small_warehouse.world:227]: yep, these values work well for both with/without GPU

File: README.md (hunk 2 above)
Comment (wangvnn) [reply README.md:71]: It is not ideal, but it is fix current broken lighting on CPU instance. We are gathering user feedback to consider proper fix for CPU/GPU rendering parity.

**Review by ojasjoshi — APPROVED** [review]

**Pull request merged**